scripts/apps/authoring/authoring/directives/sd-static-autocomplete.ts

```diff
@@ -0,0 +1,125 @@
+import {debounce} from 'lodash';
```

Comment:
> `sd-static-autocomplete.ts`

How is it static? How would a dynamic autocomplete work?

Reply:
it would query server or something on every value change, here directive works with the list it gets upfront

Reply:
If it's being autocompleted from published sluglines(or is it from another place?) it will be thousands of items in which case it would start lagging at some point. Wouldn't it be better to use a server filtered autocomplete instead?

Reply:
there is a server config to set how long in history it should look for values. usually it will check for last 1-2 days